articles/rules-are-dangerous.md

```diff
@@ -2,45 +2,45 @@
+The problems I often see in projects with such rules are as follows:
 * Applying rules in the wrong context might not have the desired effect at all
-* Using rules stricter than meant to be, causes adverse effects
-* Applying rules without really understanding them can result in more harm to the project than benefits
-* Treating rules as god given, causes people not to reason about these rules anymore. This can cause a destructive discussion culture and lead to bad decisions without real arguments.
+* Using rules more strictly than they were intended to be used causes adverse effects
+* Applying rules without really understanding them can result in more harm than good to the project 
+* Treating rules as God-given, causes people not to reason about these rules anymore. This can cause a destructive discussion culture and lead to bad decisions without real arguments.
 
 ## Recommendations
 
 Here are my recommendations on how to mitigate the problems with such rules in your projects:
 
-* We all must be always willing to explain ourselves, why we do something like that and what are the arguments to apply a rule in which context. Something like “just because our process manual says so” or “because I’ve read it in that article” is usually not a good answer.
-* Don’t try to replace critical thinking and adaptive methods by exact and rigid rules that people must follow strictly. 
-* Do not read my recommendations here in a dogmatic way. Which would be: “Rolf said rules are bad and harmful, so let’s trash all our rules in our project and don’t follow the scrum guide, our Definition of Done, Clean Code and similar things anymore”. This was not my message at all! 
-* Instead: Discuss, write down and live these rules in your team, but see it more as a guide than exact rules to be 100% precisely followed in any case.
-* Note down not just the rules you use in your project but also the arguments why a rule should be applied. 
-* Be open to discuss and reexplain rules in your projects with your team members and to adjust the rules to changes in the context (e.g. growing software, growing team, shorter release cycles, different team members, new ideas, etc.)
-* Be critical to existing rules, but also respect and try to understand them in a first place.
+* We must all be always willing to explain ourselves, why we do something in a particular way and what the arguments are for applying a rule in a particular context. Something like “just because our process manual says so” or “because I’ve read it in that article” is usually not a good answer.
+* Don’t try to replace critical thinking and adaptive methods with exact and rigid rules that people must follow strictly. 
+* Do not read my recommendations here in a dogmatic way. Doing so would be to conclude: “Rolf said rules are bad and harmful, so let’s trash all the rules in our project and not follow the scrum guide, our Definition of Done, Clean Code and similar things anymore”. This was not my message at all! 
```

Comment:
I learnt from @peitor that "Scrum" should always be written with first letter in big - like God ;-)  maybe even "Scrum Guide"??

Reply:
done https://github.com/Zuehlke/fifty-shades/commit/5a689473f22bcb2bba1abc86d809c5e37268e6eb